avocado/core/remote/runner.py

```diff
@@ -264,6 +265,13 @@ def run_suite(self, test_suite, variants, timeout=0, replay_map=None):
             self.result.end_tests()
             self.job._result_events_dispatcher.map_method('post_tests',
                                                           self.job)
+            # compare the major version of remote and local version of avocado. If different write to UI and job.log
+            if str(remote_version[0]) != version.MAJOR:
+                log = logging.getLogger("avocado.job")
+                self.job.log.info("WARNING    : avocado version on the remote system is %s, local is %s",
```

Comment:
Btw, using `APP_LOG.warn()`, you don't need the string `WARNING :`. The yellow color is enough to say it's a warning. So you can make a better use of that space. I'd use a message like `VERSIONS: Local (%s) and remote (%s) are not the same.`